### ISSUE TITLE
headnote for 1 H.6

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/xml#144]
> reicht wirklich nicht aus

**Comment by dietmarpravida** [comment]
naja, hier gibt es folgenden Eintrag:

`<headNote>1 Doppelblatt. – Zu ‚Faust I‘.</headNote>`
        `<note>Göttling an Goethe, 12. Juni 1825, mit Göttlings Korrekturen zu faust://document/faustedition/B_9 und diesbezüglichen Anmerkungen Goethes. Die Transkription wurde von Annette Mönnich (Weimar) angefertigt und für die Faustedition aufbereitet.</note>`

**Comment by thvitt** [comment]
Diese `<note>`s sollten aus Gründen unterdrückt werden. Irgendjemand (ABR?) wollte die alle mal durchschauen, [die entsprechende Übersicht existiert noch](http://dev.faustedition.net/meta/first-notes), ich habe aber nie ein Go bekommen, die wieder einzubinden.

**Comment by gerritbruening** [comment]
Ja, die sollten vorher prüfend durchgesehen und etwas vereinheitlicht werden; klassisches Problem von Freitextfeldern. Aber die `headNote`s sollten stimmen, das würde ich als erstes machen. Wenn bedeutet, Inhalte aus den `note`s hochzuziehen, finde ich das in Ordnung.
